libraries/mbed/targets/hal/TARGET_RENESAS/TARGET_RZ_A1H/pwmout_api.c

```diff
@@ -57,6 +83,31 @@ static PWMType PORT[] = {
      PWM2F,          // PWM13_PIN
 };
 
+static MTU2_PWMType MTU2_PORT[] = {
```

Comment:
Hello

Thanks your comments.
I modifed CONST declaration of the table is not enough.